pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -419,7 +419,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 					},
 					{
 						Record: "kubevirt_num_virt_handlers_by_node_running_virt_launcher",
-						Expr:   intstr.FromString("count by(node)(node_namespace_pod:kube_pod_info:{pod=~'virt-launcher-.*'} ) * on (node) group_left(pod) (1*(kube_pod_container_status_ready{pod=~'virt-handler-.*'} + on (pod) group_left(node) (0 * node_namespace_pod:kube_pod_info:{pod=~'virt-handler-.*'} ))) or on (node) (0 * node_namespace_pod:kube_pod_info:{pod=~'virt-launcher-.*'} )"),
+						Expr:   intstr.FromString("sum by (node)(0*node_namespace_pod:kube_pod_info:{pod=~'virt-launcher-.*'}) + on(node) group_left() (sum by(pod, node)(kube_pod_container_status_ready{pod=~'virt-handler-.*'} + on(pod) group_left(node) (0* node_namespace_pod:kube_pod_info:{pod=~'virt-handler-.*'}))) or on(node) ( sum by(node)(0*node_namespace_pod:kube_pod_info:{pod=~'virt-launcher-.*'}))"),
```

Comment:
Hey,
Thank you for fixing this bug ,
I'm not sure about that query but if i understand correctly it suppose to generate an metric with zero value when virt-handler isn't ready in one node and there is at least one virt-launcher pod running in the same node.(please correct me if i'm wrong)
so i tried it myself and i didn't get any result from this metric after deleting all virt-handlers from a cluster with 4 virt-launcher:
![image](https://user-images.githubusercontent.com/91669950/162197390-cbdd73d6-cf79-4a0d-8305-7a6e7d3fcfed.png)